research/minigo/minigo.py

```diff
@@ -305,38 +278,189 @@ def evaluate(trained_models_dir, black_model_name, white_model_name,
   return black_model_name if winner == go.BLACK_NAME else white_model_name
 
 
-def _set_params_from_board_size(board_size):
+def _set_params():
   """Set hyperparameters from board size."""
   params = model_params.MiniGoParams()
-  k = utils.round_power_of_two(board_size ** 2 / 3)
+  k = utils.round_power_of_two(FLAGS.board_size ** 2 / 3)
```

Comment:
Taking board size and other required params is preferable to relying on global flags here.